Add "Featured artist showcase beatmaps" wiki article (#5245)
ppy/osu-wiki: wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md

# Featured artist showcase beatmaps

Prior to the announcement of upcoming [featured artists](wiki/Featured_Artists), members of the [Mappers' Guild](/wiki/Mappers_Guild) who have either achieved Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance are being invited every so often to create exclusive "showcase beatmaps" using songs and other materials from osu!'s featured artist backlog that are yet to be released. When the time to announce the featured artists in question are due, these beatmaps will be treated to a comprehensive coverage in the corresponding [news posts](https://osu.ppy.sh/home/news) and will occasionally be promoted in the [@osugame Twitter](https://twitter.com/osugame) as well.

This article lists all the showcase beatmaps that have been created by the members of the Mappers' Guild over time.

## 2017

> ### November 2017

Reviewer: "2017" is redundant here, consider changing to just "November" (this goes for all others as well)

  - (![][o!s]) [Cranky - Rave 2 Rave](https://osu.ppy.sh/beatmapsets/751896#osu/1582902) hosted by ![][flag_KR] [Sonnyc](https://osu.ppy.sh/users/11771)
  - (![][o!s]) [Cranky - T&J](https://osu.ppy.sh/beatmapsets/752022#osu/1583147) hosted by ![][flag_SG] [handsome](https://osu.ppy.sh/users/2123087)

### August 2018

- **[High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music)** (2018-08-13)
  - ![][o!s] [Andromedik - Invasion](https://osu.ppy.sh/beatmapsets/789544#osu/1656911) hosted by ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009)
  - ![][o!s] [Gourski x Himmes - Silence](https://osu.ppy.sh/beatmapsets/789905#osu/1657683) hosted by ![][flag_FR] [BOUYAAA](https://osu.ppy.sh/users/405449)
  - ![][o!s] [James Portland - Sky](https://osu.ppy.sh/beatmapsets/789528#osu/1656874) hosted by ![][flag_SG] [handsome](https://osu.ppy.sh/users/2123087)
  - ![][o!s] [Lexurus - Gravity](https://osu.ppy.sh/beatmapsets/789529#osu/1656879) hosted by ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994)
  - ![][o!s] [Phonetic - Journey](https://osu.ppy.sh/beatmapsets/795432#osu/1670491) hosted by ![][flag_KR] [Left](https://osu.ppy.sh/users/5062061)

>   - ![][o!s] [Pulse - LP](https://osu.ppy.sh/beatmapsets/789374#osu/1656604) hosted by ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418)

Reviewer: these are mising the parentheses around the game mode icon. personally i think they look better without parentheses tho.

### November 2018

> - **[Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate)** (2018-11-21)

Author: hmm i don't think this is the case here as there are also some other articles out there such as [Mapping and modding timeline](https://osu.ppy.sh/wiki/en/Mapping_and_Modding_Timeline) which uses the same YY-MM-DD format for convenience tho :(

Even from its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists" <!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals; in fact, the six beatmaps that were made for the [MOtOLOiD featured artist announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) were also the Guild's first ever creations. 

As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were pften restricted only to a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode. However, [the Guild's shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) has gradually allowed more and more mappers to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode given that they either have achieved Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance.

> Starting from [MYUKKE.'s featured artist announcement](https://osu.ppy.sh/home/news/2020-12-05-new-featured-artist-myukke) in December 2020 onwards, new featured artist announcement posts have also began to feature short preview videos — often with these showcase beatmaps as their main headline — every now and then.

Reviewer: ```suggestion
Starting from [MYUKKE.'s featured artist announcement](https://osu.ppy.sh/home/news/2020-12-05-new-featured-artist-myukke) in December 2020 onwards, new featured artist announcement posts occasionally began to feature short preview videos, often with these showcase beatmaps acting as their main headline.
```

> [o!s]: /wiki/shared/mode/osu.png "osu!"

Reviewer 2: ```suggestion
[o!s]: /wiki/shared/mode/osu.png "osu!standard"
```

Reviewer 3: Not standard. just osu!. it's not hard.

Reviewer 4: > Not standard. just osu!. it's not hard.

The only problem is that `osu!standard` is written as such all over the place, if the writing for the `standard` game mode is indeed that one, we should think about changing it all over the place, on each article.

Reviewer 2: The ASC needs that change as well if so.

Author: ah, yeah, i've adjusted the alt texts accordingly in one of my earlier commits dw ;~;

![image](https://user-images.githubusercontent.com/36564236/114387245-67a5ad00-9bbc-11eb-9928-52cfcbdc3f69.png)

![image](https://user-images.githubusercontent.com/36564236/114387343-8015c780-9bbc-11eb-81f0-b65dd57c14ef.png)

as for the asc -- yeah it does seem that the asc needs some updating as it still promotes the use of `osu!standard` despite the fact that `osu!` has been the much preferred term for a long while. will open a separate pr on this in a bit

Reviewer 4: > as for the asc -- yeah it does seem that the asc needs some updating as it still promotes the use of `osu!standard` despite the fact that `osu!` has been the much preferred term for a long while. will open a separate pr on this in a bit

I'm looking into changing `osu!standard` to `osu!` from everywhere, I'll go update at the same time ASC.

# Featured artist showcase beatmaps

> Prior to the announcement of upcoming [featured artists](wiki/Featured_Artists), members of the [Mappers' Guild](/wiki/Mappers_Guild) who have achieved either Tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards) or been accepting certain [limited-time mystery quests](wiki/Mappers_Guild#quests) set by the Guild in advance are occasionally invited to create exclusive "showcase beatmaps" by using songs and other materials from osu!'s featured artist backlog that are yet to be released. Once it is time to publicly announce the featured artists in question, these beatmaps will be comprehensively covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and will occasionally be promoted on the [@osugame Twitter](https://twitter.com/osugame "Twitter") as well.

Reviewer 5: ```suggestion
Members of the [Mappers' Guild](/wiki/Mappers_Guild) occasionally create exclusive "showcase beatmaps" by using songs and other materials from osu!'s unreleased [featured artists](wiki/Featured_Artists). When a featured artist is publicly announced, these beatmaps are covered in the corresponding [newsposts](https://osu.ppy.sh/home/news) and are often promoted on the [@osugame Twitter](https://twitter.com/osugame).

To participate in featured artist showcase beatmaps, a member of the Mappers' Guild must accept a [limited-time mystery quests](wiki/Mappers_Guild#quests) or be directly invited to create a beatmap after reaching tier 1 (or higher) in [Guild rank](/wiki/Mappers_Guild#rewards).
```
no problem with the info, but the way the first sentence is worded is really hard to read (too many clauses)

  - (![][osu!]) [Xanthochroid - The Sound of Hunger Rises](https://osu.ppy.sh/beatmapsets/1335143#osu/2766094) hosted by ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410)

### April

- **[YUZUKINGDOM](https://osu.ppy.sh/home/news/2021-04-10-new-featured-artist-yuzukingdom)** (2021-04-10)
  - (![][osu!]) [YUZUKINGDOM - Roskill Lavender](https://osu.ppy.sh/beatmapsets/1410727#osu/2908440) hosted by ![][flag_US] [Nao Tomori](https://osu.ppy.sh/users/5364763)
- **[fiend](https://osu.ppy.sh/home/news/2021-04-14-new-featured-artist-fiend)** (2021-04-14)
  - (![][osu!]) [fiend - FEVER DREAM (feat. yzzyx)](https://osu.ppy.sh/beatmapsets/1405913#osu/2899195) hosted by ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805)

## History

Even from its very beginning, the Mappers' Guild has been set into pace with "creating promotional beatmap packs for upcoming featured artists" <!-- https://osu.ppy.sh/community/forums/topics/875743 --> as one of its primary goals; in fact, the six beatmaps that were made for the [MOtOLOiD featured artist announcement](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) were also the Guild's first ever creations. 

> As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were pften restricted to only a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode; however, the Guild's [shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) gradually allowed more and more creators to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode.

Reviewer 5: ```suggestion
As the Guild itself was operating on a closed, application-based entry in its early days, access to upcoming featured artist contents were often restricted to only a handful of mappers who were predominantly of the [osu!](wiki/Game_mode/osu!) game mode; however, the Guild's [shift to an open and automated entry system in early 2019](wiki/Mapping_and_Modding_Timeline#march.5) gradually allowed more and more creators to participate in the creation of showcase beatmaps for future featured artists over time regardless of their game mode.
```
